index.js

```diff
@@ -61,7 +61,7 @@ module.exports = function(content) {
 	}
 
 	var publicPath = "__webpack_public_path__ + " + JSON.stringify(url);
-	if (config.publicPath) {
+	if (config.publicPath || config.publicPath === "") {
```

Comment:
@dms1lva Could you try with `if (config.publicPath !== false)` like it's done in #122, I would favour this PR since it contains proper testing, but the fix in #122 is cleaner :)